onnxruntime/test/optimizer/free_dimension_override_test.cc

```diff
@@ -17,7 +17,7 @@ using namespace ONNX_NAMESPACE;
 namespace onnxruntime {
 namespace test {
 
-TEST(FreeDimensionOverrideTransformerTest, Test) {
+TEST(FreeDimensionOverrideTransformerTest, DISABLED_Test) {
```

Comment:
Do we plan to re-enabling this test?

Reply:
it should be enabled, after Jeff's sign off this change, since it impacts winml.